metadata.rb

```diff
@@ -9,3 +9,5 @@
 
 issues_url 'https://github.com/nuclearsandwich/cookbook-ros-buildfarm/issues'
 source_url 'https://github.com/nuclearsandwich/cookbook-ros-buildfarm/'
+
+depends 'jenkins'
```

Comment:
You should add a version constraint here. Probably best to stick with the current major version (so use a constraint that limits us to any 8.*) https://docs.chef.io/config_rb_metadata/#version-constraints